shell/browser/native_window_mac.h

```diff
@@ -235,9 +234,9 @@ class NativeWindowMac : public NativeWindow,
 
   ElectronNSWindow* window_;  // Weak ref, managed by widget_.
 
-  base::scoped_nsobject<ElectronNSWindowDelegate> window_delegate_;
-  base::scoped_nsobject<ElectronPreviewItem> preview_item_;
-  base::scoped_nsobject<ElectronTouchBar> touch_bar_;
+  ElectronNSWindowDelegate* __strong window_delegate_;
+  ElectronPreviewItem* __strong preview_item_;
+  ElectronTouchBar* __strong touch_bar_;
```

Comment:
This is me trying to grok ARC, so sorry if this is a dumb question - do we need to intentionally release __strong variables at any point? Will ARC handle them automatically?

Reply:
Looks like ARC would handle them, I'll resolve this 😄 https://developer.apple.com/library/archive/releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html#//apple_ref/doc/uid/TP40011226-CH1-SW16